### ISSUE TITLE
Update project to PureScript 0.13

### ISSUE THREAD

**Issue by JordanMartinez** [JordanMartinez/purescript-jordans-reference#343]
Might be a few days or even longer before all dependencies compile on this release.

**Comment by JordanMartinez** [comment]
Issues currently blocking this:
- `purescript-spec` switched to using `Aff`. While I can make that switch easily, the question becomes how to explain that monad to someone unfamiliar with it...
- `benchotron` won't compile anymore due to using an illegal escape character
- module `Type.Row` was renamed (IIRC)
- Halogen v4's `HE.input_` is having an issue (I'll have to look at it in more depth)
- MarkdownRenderer (file in Projects folder's ToC folder) has an indentation issue

Overall, not too bad!

**Comment by JordanMartinez** [comment]
So, I've updated most of the Halogen code to v5 (#320). However, the query API works differently now. Before, it returned `a`, but now it returns `Maybe a`. In the random number game, where I was experimenting with running the business logic in one monad and using an AVar to block the business logic from running until after the view was updated and got the user input, this API change somewhat forces me to redesign parts of the code in ways that breaks assumptions I did not know I was making.

As a result, that is bottlenecking this issue (though I havne't yet looked into benchotron and the purescript-spec update.
